terraform/modules/postgrest/main.tf

```diff
@@ -0,0 +1,40 @@
+resource "google_cloud_run_v2_service" "postgrest" {
+  depends_on = [var.migrations_complete]
+  location   = var.region
+  name       = "postgrest"
+  template {
+    containers {
+      image = "postgrest/postgrest:v11.2.1"
+      env {
+        name  = "PGRST_DB_ANON_ROLE"
+        value = "web_anon"
+      }
+      env {
+        name  = "PGRST_DB_MAX_ROWS"
+        value = var.postgrest_max_rows
+      }
+      env {
+        name  = "PGRST_DB_URI"
+        value = var.db_conn_str_private
+      }
+      ports {
+        container_port = 3000
+      }
+    }
+    scaling {
+      min_instance_count = 1
+      max_instance_count = 1
```

Comment:
Since this will be public, are we sure we want to prohibit more than one instance?

Reply:
I guess we can leave this as is and let people update this according to their need, since this might be project dependent. People would fork and keep their own version of Inbox just like we do with the Aptos processor.